Button.css

```diff
@@ -0,0 +1,36 @@
+.btn {
+    margin : 1.25rem;
+    font-family: "comic sans ms";
+    font-weight: 50%;
+    width : 12.5rem;
+    height : 3.125rem;
+    font-size: 1.125rem;
+    background-color: aqua;
+    
+    border : 0.125rem solid white;
+    border-radius: 1.25rem;
+
+}
+
+.txt-style{
+    color : rgb(0, 0, 0);
+    text-decoration: none;
+}
+
+.btn-link {
```

Comment:
Just a suggestion like you can pass this border: none and text-decoration: none properties to the reset CSS which will be applied to all button and a tag then you don't have to specify every time.
Everything else is good to go.

Reply:
noted